### PR TITLE
upgrade app to run with ruby 2.5

### PR DESCRIPTION
**Pull request by kares**
as 2.3 (even 2.4) are EOL and no longer supported

with the new Ruby version a gem update was needed (oj was using old deprecated native ext API)
\+ rubocop needs to be newer -> means it will bring in more cops on us, and thus the noisy parts ...

also I am not sure about the Docker image, since I can not access the registry.

*NOTE: this PR assumes https://github.com/CruGlobal/mobile-content-api/pull/185 merged first (to avoid conflicts)*

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM 056154071827.dkr.ecr.us-east-1.amazonaws.com/base-image-ruby-version-arg:2.3
+FROM 056154071827.dkr.ecr.us-east-1.amazonaws.com/base-image-ruby-version-arg:2.5
```
Comment (kares) [inline Dockerfile]: @Omicron7 not sure about the Docker image (can not access the registry), is this enough?

**Comment by frett** [comment]
@kares you will need to address the merge conflicts

**Comment by kares** [comment]
@frett yy, was waiting for @Omicron7 to confirm the registry URL

**Comment by frett** [comment]
the registry URL is correct, it fully built and deployed to Docker in AWS

File: Dockerfile (hunk 1 above)
Comment (Omicron7) [reply Dockerfile]: Yeah, that works.

**Pull request merged**